src/index.js

```diff
@@ -341,6 +341,11 @@ async function jsPDFInvoiceTemplate(props) {
     compressPdf: true,
   };
 
+  const isBreakPage = (height, pageHeight) => {
```

Comment:
@senukavee98 can u add comment here whats this for also why we are doing this `height > pageHeight - 10`?